draft-ietf-quic-transport.md

```diff
@@ -4850,6 +5697,61 @@ MUST treat a failed validation as a connection error of type
 PROTOCOL_VIOLATION.
 
 
+#### Retry Packet Integrity {#retry-integrity}
```

Comment:
A painful reminder that Retry is one of the few sections that uses a fourth-level heading, but I can't fault this placement.